src/components/views/beacon/RoomCallBanner.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Fragment, useCallback } from "react";
+import { MatrixEvent, Room } from "matrix-js-sdk/src/matrix";
+
+import { _t } from "../../../languageHandler";
+import AccessibleButton, { ButtonEvent } from "../elements/AccessibleButton";
+import dispatcher, { defaultDispatcher } from "../../../dispatcher/dispatcher";
+import { ViewRoomPayload } from "../../../dispatcher/payloads/ViewRoomPayload";
+import { Action } from "../../../dispatcher/actions";
+import { Call, ConnectionState, ElementCall } from "../../../models/Call";
+import { useCall } from "../../../hooks/useCall";
+import { RoomViewStore } from "../../../stores/RoomViewStore";
+import { useEventEmitterState } from "../../../hooks/useEventEmitter";
+import {
+    OwnBeaconStore,
+    OwnBeaconStoreEvent,
+} from "../../../stores/OwnBeaconStore";
+import { CallDurationFromEvent } from "../voip/CallDuration";
+import { MockedCall } from "../../../../test/test-utils/call";
+
+interface RoomCallBannerProps {
+    roomId: Room["roomId"];
+    call: Call;
+}
+
+const RoomCallBannerInner: React.FC<RoomCallBannerProps> = ({
+    roomId,
+    call,
+}) => {
+    let callEvent: MatrixEvent = null;
```

Comment:
```suggestion
    let callEvent: MatrixEvent | null = null;
```

to make strict mode happy